src/js/tracks/text-track-settings.js

```diff
@@ -235,113 +235,112 @@ function setSelectedOption(target, value) {
-              <option value="#0FF">Cyan</option>
-            </select>
-            <span class="vjs-text-opacity vjs-opacity">
-              <select>
-                <option value="">---</option>
-                <option value="1">Opaque</option>
-                <option value="0.5">Semi-Opaque</option>
+         <div class="vjs-tracksettings-colors">
+          <fieldset>
+            <legend>Colors</legend>
+            <div class="vjs-fg-color vjs-tracksetting">
+              <label class="vjs-label" for="captions-foreground">Foreground</label>
+              <select id="captions-foreground">
+                <option value="#FFF" selected>White</option>
+                <option value="#000">Black</option>
+                <option value="#F00">Red</option>
+                <option value="#0F0">Green</option>
+                <option value="#00F">Blue</option>
+                <option value="#FF0">Yellow</option>
+                <option value="#F0F">Magenta</option>
+                <option value="#0FF">Cyan</option>
               </select>
-            </span>
-        </div> <!-- vjs-fg-color -->
-        <div class="vjs-bg-color vjs-tracksetting">
-            <label class="vjs-label">Background</label>
-            <select>
-              <option value="">---</option>
-              <option value="#FFF">White</option>
-              <option value="#000">Black</option>
-              <option value="#F00">Red</option>
-              <option value="#0F0">Green</option>
-              <option value="#00F">Blue</option>
-              <option value="#FF0">Yellow</option>
-              <option value="#F0F">Magenta</option>
-              <option value="#0FF">Cyan</option>
-            </select>
-            <span class="vjs-bg-opacity vjs-opacity">
+              <span class="vjs-text-opacity vjs-opacity">
+                <select>
+                  <option value="1" selected>Opaque</option>
+                  <option value="0.5">Semi-Opaque</option>
+                </select>
+              </span>
+            </div>
+            <div class="vjs-bg-color vjs-tracksetting">
+              <label class="vjs-label" for="captions-background">Background</label>
+              <select id="captions-background">
+                <option value="#000" selected>Black</option>
+                <option value="#FFF">White</option>
+                <option value="#F00">Red</option>
+                <option value="#0F0">Green</option>
+                <option value="#00F">Blue</option>
+                <option value="#FF0">Yellow</option>
+                <option value="#F0F">Magenta</option>
+                <option value="#0FF">Cyan</option>
+              </select>
+              <span class="vjs-bg-opacity vjs-opacity">
                 <select>
-                  <option value="">---</option>
                   <option value="1">Opaque</option>
+                  <option value="0.8" selected>80% Opaque</option>
```

Comment:
is adding another bg opacity necessary? Would it be better to change the value for `semi-transparent` instead?
The FCC rules for this call for `opaque`, `semi-transparent`, and `transparent` options.
@OwenEdwards does WCAG or a11y have anything to say about these values?